overlord/overlord_test.go

```diff
@@ -691,8 +698,34 @@ func (ovs *overlordSuite) TestEnsureLoopPruneDoesntAbortShortlyAfterStartOfOpera
 	c.Check(chg.Status(), Equals, state.DoingStatus)
 }
 
+type ticker struct {
```

Comment:
should these be defined earlier? they are used before

Reply:
Done.